google/ggrc-core
# GGRC-8845 Replace can-map with can-define-map for Deferred Mapper component

# Issue description

Replace usage of legacy package.

# Steps to test the changes

1. Open Edit modal on Audit
2. Add some reference urls
3. Save audit
4. Expected: Added reference urls shown

# Solution description

Replace can-map with can-define-map for Deferred Mapper component

# Sanity checklist




# PR Review checklist
